addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/connector/RFXComTcpConnector.java

```diff
@@ -39,6 +39,7 @@
     public void connect(RFXComBridgeConfiguration device) throws IOException {
         logger.info("Connecting to RFXCOM at {}:{} over TCP/IP", device.host, device.port);
         socket = new Socket(device.host, device.port);
+        socket.setSoTimeout(100); // In ms. Small values mean faster shutdown but more cpu usage.
```

Comment:
It seems to work different from what was assumed, if you set the [ssoTimeout](http://docs.oracle.com/javase/7/docs/api/java/net/Socket.html#setSoTimeout%28int%29) it will throw an `SocketTimeoutException` which will imediately terminate the whole exercise so we effectively broken the support for using this `RFXComTcpConnector`